src/patternfly/components/Breadcrumb/breadcrumb.scss

```diff
@@ -84,3 +84,9 @@
     font-weight: var(--pf-global--FontWeight--semi-bold);
   }
 }
+
+// Hide divider of first child
+.pf-c-breadcrumb__item:first-child .pf-c-breadcrumb__item-divider {
```

Comment:
Is it guaranteed that all of the `.pf-c-breadcrumb__item`'s will always be siblings of one another? As in a person won't wrap that element in some other thing? If I'm reading https://github.com/patternfly/patternfly-react/issues/4422 properly, seems like that is part of the problem - `.pf-c-breadcrumb__item` isn't a direct descendent of `.pf-c-breadcrumb`.

Reply:
It is a direct descendant in the HTML. It is not in the React node hierarchy so the `React.cloneElement` hack I mentioned last week doesn't work :(

If there's a better way to write the selector I'm all for it but ill equipped... are there any good resources to play around with selectors?